Doc/library/curses.rst

```diff
@@ -175,17 +183,17 @@ The module :mod:`curses` defines the following functions:
 
 .. function:: erasechar()
 
-   Return the user's current erase character.  Under Unix operating systems this
+   Return the user's current erase character as a one-byte byte string.  Under Unix operating systems this
    is a property of the controlling tty of the curses program, and is not set by
    the curses library itself.
 
 
 .. function:: filter()
 
    The :func:`.filter` routine, if used, must be called before :func:`initscr` is
-   called.  The effect is that, during those calls, :envvar:`LINES` is set to 1; the
-   capabilities clear, cup, cud, cud1, cuu1, cuu, vpa are disabled; and the home
-   string is set to the value of cr. The effect is that the cursor is confined to
+   called.  The effect is that, during those calls, :envvar:`LINES` is set to ``1``; the
+   capabilities **clear**, **cup**, **cud**, **cud1**, **cuu1**, **cuu**, **vpa** are disabled; and the **home**
+   string is set to the value of **cr**. The effect is that the cursor is confined to
```

Comment:
I wanted to distinguish them from other names. These names are not the names of arguments (which are rendered as italic). They are the names of terminal capabilities. If you prefer, I can change them to be rendered as fixed-width (\`\`clear\`\`), but not as italic.